scripts/o3de/ExportScripts/export_utility.py

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import sys
+
+from typing import List
+
+def process_level_names(ctx, seedfile_paths, level_names, platform):
+    for level in level_names:
+        seedfile_paths.append(ctx.project_path / f'Cache/{platform}/levels' / level.lower() / (level.lower() + ".spawnable"))
```

Comment:
Can we add a couple of other things in this method?
1. Make sure that the level names actually exist in the levels folder.
2. Make sure that we don't add duplicate '.spawnable' files in the 'seedfile_paths' array, if the intention is to add/update the list.

Reply:
A small comment of what the purpose of this method would also be helpful